index.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+const dictionary = require('./res/wordlist.json')
+
+function randomBytes (length) {
+  const buf = Buffer.alloc(length)
```

Comment:
This was a fallback for tests in case Keythereum implementation failed.

Keythereum is using the [crypto api](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) for [strong entropy here](https://github.com/ethereumjs/keythereum/blob/10a067d79a751539d74b2732c799189252f6048a/lib/scrypt.js#L11669-L11686), it should work in chrome/firefox/edge.